scripts/pin_the_slow_way.py

```diff
@@ -63,8 +63,8 @@
 # compatible with 5.0.1 (the other way around would be true).
 pinned = {
           'arpack': 'arpack 3.5.*',  # NA
-          'boost': 'boost 1.65.1',  # 1.61.0
-          'boost-cpp': 'boost-cpp 1.65.1',  # NA
+          'boost': 'boost 1.66.*',  # 1.61.0
```

Comment:
Since we know that boost is bumping soname for patch versions, shouldn't this be `boost 1.66` instead?

Reply:
Ok, done in #508